### PR TITLE
Add unittests and fix some bugs during unittests

### PR DESCRIPTION
**Pull request by hooshooo**
### Why:
I should have made this PR together with https://github.com/slacgismo/SCRIPT/pull/5. However, https://github.com/slacgismo/SCRIPT/pull/5 has too many commets and I decide to create a new PR for unittests.
**Please REVIEW and MERGE https://github.com/slacgismo/SCRIPT/pull/5 before merging this PR.**

### What:
1. add unittests for currently used APIs
2. fix some bugs of routers and models found during unittests

### Checklist
- [ ] Finished My Changes
- [x] Automated Unit Tests

### CODE REVIEW

File: webserver/script/models/enums.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    # [TODO] add more connector types
+
+
+class VehicleMake(EnumWithChoices):
+    """Vehicle make"""
+    Nissan = 'Nissan'
+    Chevrolet = 'Chevrolet'
+    Audi = 'Audi'
+    BMW = 'BMW'
+    Honda = 'Honda'
```
Comment (jongoncalves) [inline webserver/script/models/enums.py]: Make the enum keys all consistent -> uppercase

**Review by jongoncalves — COMMENTED** [review]
PR is looking really good. I think we should be able to merge it really soon!

**Comment by hooshooo** [comment]
Thank you for your review.

**Review by jongoncalves — APPROVED** [review]

**Comment by jongoncalves** [comment]
I think this is ready to go

**Pull request merged**